packages/editor/src/components/hierarchy/HierarchyPanelContainer.tsx

```diff
@@ -380,18 +340,15 @@ function HierarchyPanelContents({ rootEntityUUID }: { rootEntityUUID: EntityUUID
     handleClose()
 
     const nodes = node.selected ? getState(SelectionState).selectedEntities : [node.entity]
-    copyPasteHelper
-      .copyNodes(nodes)
-      .then(() =>
-        NotificationService.dispatchNotify(t('editor:hierarchy.copy-paste.copied-node'), { variant: 'success' })
-      )
+    CopyPasteFunctions.copyEntities(nodes).then(() =>
+      NotificationService.dispatchNotify(t('editor:hierarchy.copy-paste.copied-node'), { variant: 'success' })
```

Comment:
i think dispatching a notification every time you copy something is a bit unnecessary